composer.json

```diff
@@ -25,13 +25,18 @@
         "cweagans/composer-patches": "1.6.7",
         "drupal/core-recommended": "8.8.2",
         "drush/drush": "9.7.1",
-        "drupal/bootstrap_barrio": "4.22"
+        "drupal/bootstrap_barrio": "4.22",
+        "drupal/cas": "1.5.0",
+        "drupal/externalauth": "1.2.0"
     },
     "extra": {
       "patches": {
           "drupal/bootstrap_barrio": {
               "missing theme config schema": "https://www.drupal.org/files/issues/2019-11-15/improve-theme-settings-remove-deprecated-code-3037780-22.patch"
           }
+      },
+      "installer-paths": {
+        "modules/contrib/{$name}": ["type:drupal-module"]
```

Comment:
I think we want to rely on the `installer-paths` config in the scaffolding repo and not add anything to this repo's composer.json.